README.md

```diff
@@ -136,6 +136,12 @@ An array with fixtures (defaults). Each fixture has to be an object with a `labe
 
 You can also add a `className` key to a fixture. This class will be applied to the fixture item.
 
+### maxFixtures
+Type: `Number`
+Default: `10`
+
+Maximum number of fixtures to render.
```

Comment:
It means 10 are rendered when nothing is entered. You can easily narrow down the list by typing something in the input.